### DISCUSSION TITLE
invalid slack link?

### DISCUSSION THREAD

**Discussion by woodgear** [apache/apisix#4932] [Q&A] [answered]
invalid slack link?

**Comment by bzp2010** [comment]
Hi, @woodgear.
Where did you find the wrong slack link?

**Comment by moonming** [comment]
https://join.slack.com/t/the-asf/shared_invite/zt-nggtva4i-hDCsW1S35MuZ2g_2DgVDGg
join #apisix on our Slack to meet the team and ask questions

**Reply by woodgear** [reply]
![image](https://user-images.githubusercontent.com/16863223/131441226-ab0d8f8b-193c-4df1-b7d8-6c690dbbba98.png)
use continue with google and this error popup. 
did i have to have a @apache.org email?

**Reply by moonming** [reply]
@juzhiyuan please take a look. Is this link expired?

**Reply by juzhiyuan** [reply]
It seems that Slack has changed its policy, from `never expired` to `6 days` 🤔 

![image](https://user-images.githubusercontent.com/2106987/131450698-2ff32d3a-8c06-4d7d-99ca-41148e5179fe.png)

**Reply by juzhiyuan** [reply]
@woodgear Please use this one https://join.slack.com/t/the-asf/shared_invite/zt-u54pmqj8-qy~RC9ZfM8MD74b6~tn3lA

**Reply by woodgear** [reply]
this link works.

**Comment by juzhiyuan** [comment]
I just read through the ASF's slack policy, see https://infra.apache.org/slack.html 

![image](https://user-images.githubusercontent.com/2106987/131451209-e36cf667-540d-4b30-81cc-161ee94928bc.png)

**Comment by juzhiyuan** [comment, marked as answer]
For users who don't have an Apache account, please Subscribe[1] to Apache APISIX's mailing list, and sent mail for the invitation link.

[1] https://apisix.apache.org/docs/general/subscribe-guide

**Comment by juzhiyuan** [comment]
For people who would like to join the Slack channel, you could use [https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ) this link now :)

But I'm not sure when will this link gets expired again 🤔
